docs: Small addition in add new databases with docker

JavierLopezT opened the pull request:
Based on my own noob experience

Since it is a small doc change, I think it doesn't need an Issue first

JavierLTPromofarma committed:
Small addition add new databases docker

mvoitko commented on docs/src/pages/docs/Connecting to Databases/docker-add-drivers.mdx:
````diff
@@ -58,6 +58,11 @@ After the rebuild is complete, which make take a few minutes, relaunch:
 ```
 docker-compose up
 ```
+If with the previous command, your docker gets up but Superset gets stuck loading the welcome screen, you can try with
````
It's not trial and errors approach. There are different docker-compose files for a reason. If there is a problem with `docker-compose.yml` we should address that problem not trying non-dev docker-compose.

srinify replied:
Since no reply from OP, I incorporated your feedback :)

srinify committed:
Update docker-add-drivers.mdx

Since no reply from @JavierLopezT I've gone ahead and incorporated @mvoitko's feedback and using this as an opportunity to educate (instead of showcasing an escape valve)
